NoSoft-SA/nspack
# Feature/depot returns

Summary of this change:
Depot returns
New client rule: enable_depot_returns (only true for Kromco)

If new client rule: enable_depot_returns is true THEN
show checkbox (false by default) on loads

if load.depot_return Then:
allow allocation and shipping of failed govt_inspection pallets (must be inspected pallets)

Such loads should be shown in red

CHANGELOG entry:

### Added
- [Add feature: Depot returns](https://github.com/NoSoft-SA/nspack/pull/445/commits/61aefae032fdec330e2e64a87405d0fb22526eb3)

How to test this code via the UI:

set client setting, enable_depot_returns
create new load, see new check box, depot return
